backend/pkg/config/config.go

```diff
@@ -34,12 +33,14 @@ type Config struct {
 	MetricsNamespace string `yaml:"metricsNamespace"`
 	ServeFrontend    bool   `yaml:"serveFrontend"` // useful for local development where we want the frontend from 'npm run start'
 
-	Console  Console        `yaml:"console"`
-	Redpanda Redpanda       `yaml:"redpanda"`
-	Connect  Connect        `yaml:"connect"`
-	REST     Server         `yaml:"server"`
-	Kafka    Kafka          `yaml:"kafka"`
-	Logger   logging.Config `yaml:"logger"`
+	Console        Console      `yaml:"console"`
+	Redpanda       Redpanda     `yaml:"redpanda"`
+	Connect        KafkaConnect `yaml:"kafkaConnect"`
```

Comment:
Minor... naming is hard... Should we call the actual variable something else too?